packages/restlessness-cli/rln-backend/src/models/Endpoint/templates/index.ts

```diff
@@ -1,9 +1,11 @@
 import { Authorizer, Endpoint } from 'root/models';
+import { AuthorizerEntry } from 'root/models/Authorizer';
+import {EndpointEntry} from 'root/models/Endpoint'
 
 const indexTemplate = (
   hasPayload: boolean,
   vars: string[],
-  authorizer: Authorizer,
+  authorizer: AuthorizerEntry,
```

Comment:
For the same reason, I would not consider an AuthorizerEntry, but an Authorizer class one